### PR TITLE
Composer API - add support for service enable/disable

### PR DESCRIPTION
**Pull request by ygalblum**
The Composer API supports only a subset of the customization options supported by the Blueprint file. This PR comes to add the Blueprint ability to enable/disable services to the Composer API

This pull request includes:

- [x] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: internal/cloudapi/v2/openapi.v2.yml [hunk 1]
```diff
@@ -850,6 +850,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        services:
+          type: object
```
Comment (ondrejbudai) [inline internal/cloudapi/v2/openapi.v2.yml:854]: Let's add

`additionalProperties: false`

?

File: internal/cloudapi/v2/openapi.v2.yml [hunk 2]
```diff
@@ -850,6 +850,19 @@ components:
...
+          properties:
+            enabled:
+              type: array
```
Comment (ondrejbudai) [inline internal/cloudapi/v2/openapi.v2.yml:857]: `minItems: 1`?

File: internal/cloudapi/v2/openapi.v2.yml [hunk 3]
```diff
@@ -850,6 +850,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        services:
+          type: object
+          properties:
+            enabled:
```
Comment (ondrejbudai) [inline internal/cloudapi/v2/openapi.v2.yml:856]: a small `description` would be nice I think

File: internal/cloudapi/v2/openapi.v2.yml [hunk 4]
```diff
@@ -850,6 +850,19 @@ components:
...
+              type: array
+              items:
+                type: string
+                example: "nftables"
+            disabled:
```
Comment (ondrejbudai) [inline internal/cloudapi/v2/openapi.v2.yml:861]: Same here, a `description` and `minItems` would be nice I think.

**Review by ondrejbudai — COMMENTED** [review]
Great! I've recently realized that our API specification isn't self-documenting enough and it would benefit from having more descriptions and safe-guards. WDYT? I put my suggestions inline.

File: internal/cloudapi/v2/openapi.v2.yml (hunk 1 above)
Comment (ygalblum) [reply internal/cloudapi/v2/openapi.v2.yml:854]: Done

File: internal/cloudapi/v2/openapi.v2.yml (hunk 2 above)
Comment (ygalblum) [reply internal/cloudapi/v2/openapi.v2.yml:859]: Done

File: internal/cloudapi/v2/openapi.v2.yml (hunk 3 above)
Comment (ygalblum) [reply internal/cloudapi/v2/openapi.v2.yml:857]: Done

File: internal/cloudapi/v2/openapi.v2.yml (hunk 4 above)
Comment (ygalblum) [reply internal/cloudapi/v2/openapi.v2.yml:864]: Done

**Review by ondrejbudai — APPROVED** [review]

**Pull request merged**